src/components/Modals/Onboarding/OnboardingBasic.vue

```diff
@@ -99,7 +99,8 @@ export default Vue.extend({
     const majorAcronyms = [...this.onboardingData.major];
     const minorAcronyms = [...this.onboardingData.minor];
     if (majorAcronyms.length === 0) majorAcronyms.push('');
-    if (minorAcronyms.length === 0) minorAcronyms.push('');
+    if (minorAcronyms.length === 0) minorAcronyms.push('', '');
+    if (minorAcronyms.length === 1) minorAcronyms.push('');
```

Comment:
What does this do?

Reply:
![image](https://user-images.githubusercontent.com/33106747/112555638-f2a23d00-8d9e-11eb-95c9-31a929610ed5.png)
wasnt sure if designs wanted two minor fields by default

Reply:
oh sorry this looks like a mistake. Should just be one